### PR TITLE
sdk_automl_tabular_regression_online_bq

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Boilerplate changes.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb [hunk 1]
```diff
@@ -116,39 +116,6 @@
         "to generate a cost estimate based on your projected usage."
```
Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:116]: <p>This should be using Vertex SDK for Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:116]: <pre class="ql-syntax" spellcheck="false">Line #2.    models = aiplatform.Model.list(filter="display_name=gsod_unique")
</pre><p>This won't work. Save the display_name to a variable and pass that in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Comment by ivanmkc** [comment]
Hi @soheilazangeneh
For the "Review model evaluation scores" section. 

They are using the GAPIC SDK, can you link them to the code snippet for Vertex SDK?

File: notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb (hunk 1 above)
Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:116]: <pre class="ql-syntax" spellcheck="false">Line #1.    # BUCKET_URI = "gs://your-bucket-name-unique"   # @param {type:"string"}
</pre><p>Remove commented code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:116]: <pre class="ql-syntax" spellcheck="false">Line #3.        model_display_name="gsod_unique",
</pre><p>Save display_name to a variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:116]: <pre class="ql-syntax" spellcheck="false">Line #2.    models = aiplatform.Model.list(filter="display_name=gsod_unique")
</pre><p>Use variable here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (soheilazangeneh) [reply notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb:116]: <p>You could get model evaluations results using <code>model.list_model_evaluations()</code></p><p>As an example: </p><pre class="ql-syntax" spellcheck="false">if len(model.list_model_evaluations()) &gt; 0:
   # Parse evaluation data
   model_evaluations = 
        model.list_model_evaluations([0].to_dict()

   evaluation_metrics = model_evaluations["metrics"]


</pre>

**Review by ivanmkc — APPROVED** [review]
LGTM

**Pull request merged**